build.gradle

```diff
@@ -83,6 +83,9 @@ buildscript {
         repositories {
             google()
             mavenCentral()
+            maven {
```

Comment:
Not relevant with the fix. But I need it if we are going to share the jfrog repo.